client/python/kserve-api/samples/grpc_infer_resnet.py

```diff
@@ -52,7 +52,7 @@
                         dest='model_name')

```

Comment:
KFS prefix actually makes those files easier to find when searching examples. @krzyczar was actually praising this that we could follow this convention with tfs samples.

Reply:
Isn't the location of those samples enough - `.../kserve-api/samples`? 
Adding `kfs` prefix makes all examples names longer and if we choose to do that we should probably also change names of the samples for TFS and ovmsclient.